### PR TITLE
Tidy up computer construction and deconstruction

### LINKED ISSUE #3669: Console construction lacks messages for starting each step
When you're building a console, each step will only display a message when it successfully finishes, unlike everything else which gives you messages such as "You begin adding cables to the frame... You add cables to the frame."
Some steps even play sounds, they simply lack a starting message.

To reproduce: Build a console frame from 5 metal sheets, do the thing, it no good

### PR DESCRIPTION
**Pull request by ghost**
Yes I'm not using construction datums, but the framework will be solid enough after this PR to be turned into one if anyone can bother
- Add bitflags and basic busy checks to avoid resource duplication
- Construction steps take the same time as their respective deconstruction steps
- Span, visible_message, formatting, newline

Fixes #3669

### CODE REVIEW

File: code/game/machinery/computer/buildandrepair.dm [hunk 1]
```diff
@@ -266,128 +271,149 @@
...
-					qdel(src)
+				if(WT.remove_fuel(0))
+					playsound(get_turf(src), 'sound/items/Welder.ogg', 50, 1)
+					user.visible_message("<span class='warning'>[user] starts dismantling \the [src].</span>", \
+					"<span class='notice'>You start dismantling \the [src].</span>")
+					if(do_after(user, 40))
+						user.visible_message("<span class='warning'>[user] dismantles \the [src].</span>", \
+						"<span class='notice'>You dismantle \the [src].</span>")
+						getFromPool(/obj/item/stack/sheet/metal, get_turf(src), 5)
+						returnToPool()
```
Comment (eswordthecat) [inline code/game/machinery/computer/buildandrepair.dm]: It's returnToPool(object)

Comment (dylanstrategie) [inline code/game/machinery/computer/buildandrepair.dm]: Right, shit, give me a second

**Comment by dylanstrategie** [comment]
I know everyone loves construction datums and such, but this fixes an issue, so yeah

**Comment by dylanstrategie** [comment]
Fug

**Comment by 9600bauds** [comment]
But why was this closed

**Pull request closed without merging**